gnomad/utils/generic.py

```diff
@@ -1469,10 +1469,11 @@ def subset_samples_and_variants(
             hl.agg.any(mt[gt_expr].is_non_ref()) | hl.agg.any(hl.is_defined(mt.END))
         )
     else:
-        mt = mt.filter_rows((hl.agg.any(mt[gt_expr].is_non_ref())) > 0)
+        mt = mt.filter_rows(hl.agg.any(mt[gt_expr].is_non_ref()))
 
     logger.info(
         f"Finished subsetting samples. Kept {mt.count_cols()} "
         f"out of {full_count} samples in MT"
     )
+    
     return mt
```

Comment:
sorry, my bad here, I meant new line at the end of the file or else git makes that little angry 🚫  thing. Otherwise looks good to me!